.github/retag-and-push.sh

```diff
@@ -52,6 +36,34 @@ process_tagged_version() {
     done
 }
 
+name=$(clean_string "${DOCKER_REGISTRY}/${project}")
+input_image="${project}:${image_tag}"
+
+if [[ -n "${NEW_TAG}" ]]; then
+    if is_valid_tag_name ${NEW_TAG} ; then
+        # push `semver` tagged image
+        semver="${NEW_TAG/v/}"
+        echo "Detected Tag: ${semver}"
+        process_tagged_version ${semver}
```

Comment:
Do we want to `exit 0` here? ... If not - what is the reason to execute the code below? (sorry for the dumb question)